golem/task/taskmanager.py

```diff
@@ -651,6 +651,7 @@ def sign_task_header(self, task_header):
         task_header.sign(private_key=self.keys_auth._private_key)  # noqa pylint: disable=protected-access
 
     def verify_subtask(self, subtask_id):
+        logger.debug("verify_subtask: %s", subtask_id)

```

Comment:
https://github.com/golemfactory/golem-internal/wiki/Logging-guidelines#log-message-structure
```suggestion
         logger.debug("verify_subtask. subtask_id=%r", subtask_id)
```